get-started/index.md

```diff
@@ -12,24 +12,25 @@ Cryostat {{ site.data.versions.cryostat.version }}
 {:toc}
 
 ## [Installing Cryostat Operator](#installing-cryostat-operator)
-Follow the steps below to install the Cryostat Operator via [OperatorHub](https://operatorhub.io/operator/cryostat-operator).
+Follow the steps below to install the **Cryostat Operator** via [**OperatorHub**](https://operatorhub.io/operator/cryostat-operator).
 
 ### [Install cert-manager](#install-cert-manager)
-The Cryostat Operator requires [cert-manager](https://cert-manager.io/) v{{ site.data.versions.cert-manager.version }}+ to run.
+
+The **Cryostat Operator** requires [cert-manager](https://cert-manager.io/) v{{ site.data.versions.cert-manager.version }}+ to run.
```

Comment:
I'm thinking that if we're highlighting (bolding) **Operator Lifecycle Manager** as a term, then **cert-manager** probably deserves bolding, too. They both have a similar relationship to Cryostat/Cryostat Operator.